util/src/main/scala/com/scalableminds/util/image/Color.scala

```diff
@@ -21,6 +22,14 @@ object Color {
     )
   }
 
+  def fromHTML(htmlCode: String): Color = {
```

Comment:
This should probably be wrapped in a tryo (possibly with toOption). The usage could then use the default if no valid color is parsed.